### PR TITLE
[FIX] website_payment: correct compute of providers state

### PR DESCRIPTION
**Pull request by alt-odoo**
In case stripe provider was deleted, the value of 'stripe' is None. This will trigger a TypeError using the 'in' operation. We should adapt the conditional statement to properly do the computation.

Description of the issue/feature this PR addresses:
opw-3057699
opw-3057724
opw-3046994
opw-3047401

Current behavior before PR:
Impossible to access Settings if stripe provider was deleted.

Desired behavior after PR is merged:
Correct access to Settings even if stripe provider was deleted.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by alt-odoo** [comment]
Coucou @AntoineVDV 

Pourrais-tu review ceci stp ? Ca devrait aller vite :)

**Review by AntoineVDV — COMMENTED** [review]
Hello @alt-odoo 

LGTM but if you're trying to reach the team responsible for that module that'd rather be @odoo/rd-ecommerce-documents (guys you might want to update your codeowner rules) :wink:

**Review by Williambraecky — APPROVED** [review]
LGTM thanks

**Comment by alt-odoo** [comment]
@Williambraecky 

Could you r+ it now everything is green and approved ?
Thanks !

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**
